src/app/business/block/file-share-detail/file-share-detail.component.ts

```diff
@@ -352,11 +352,22 @@ export class FileShareDetailComponent implements OnInit{
         this.SnapshotService.deleteSnapshot(id).subscribe((res)=>{
             this.getSnapshots(this.fromFileShareId);
             this.showCreateSnapshot = true;
+            this.msgs = [];
+            this.msgs.push({severity: 'success', summary: 'Success', detail: "Fileshare snapshot will be deleted shortly."});
+        }, (err)=>{
+            this.msgs = [];
+            this.showCreateSnapshot = true;
+            this.msgs.push({severity: 'error', summary: 'Error', detail: err.message ? err.message : err.json().message});
         })
     }
     deleteAcls(aclId){
         this.FileShareAclService.deleteFileShareAcl(aclId).subscribe((res)=>{
             this.getAcls();
+            this.msgs = [];
+            this.msgs.push({severity: 'success', summary: 'Success', detail: "Fileshare ACL will be deleted shortly."});
```

Comment:
Fileshare ACL ? I think the message is not proper?

Reply:
This is for deleting File share access. Do we call it "fileshare access"?
@kumarashit @NajmudheenCT